decidim_app-design/app/views/public/partials/_comments.html.erb

```diff
@@ -154,93 +170,69 @@
           </a>
         </div>
       </div>
-      <article class="comment comment--highlight comment--nested">
-        <div class="comment__header">
-          <div class="author-data">
-            <div class="author-data__main">
-              <div class="author author--inline">
-                <a href="#" class="author__avatar">
-                  <%= image_tag "demo-avatar.jpg" %>
-                </a>
-                <a href="#" class="author__name">
-                  Ajuntament de Barcelona
+      <div class="add-comment add-comment--reply" id="comment2-reply"
+           data-toggler=".is-active">
+        <form>
+          <label class="show-for-sr" for="add-comment-2">Resposta</label>
+          <textarea id="add-comment-2"
+            placeholder="Respon a aquest comentari"></textarea>
+          <input type="submit" class="button small hollow" value="Enviar">
+        </form>
+      </div>
+      <div class="hide" id="comments-level-1" data-toggler=".hide">
+        <article class="comment comment--highlight comment--nested">
+          <div class="comment__header">
+            <div class="author-data">
+              <div class="author-data__main">
+                <div class="author author--inline">
+                  <a href="#" class="author__avatar">
+                    <%= image_tag "demo-avatar.jpg" %>
+                  </a>
+                  <a href="#" class="author__name">
+                    Ajuntament de Barcelona
+                  </a>
+                  <time datetime="2016-08-01T19:47Z">1-08-2016 21:47</time>
+                </div>
+              </div>
+              <div class="author-data__extra">
+                <%= partial "flag_empty" %>
+                <a href="comment-view#single-comment" title="Enlace permamente">
```

Comment:
The link doesn't work in some cases. I think is due to the lack of a comment view detail of other entities like initiatives.
Would It be better to force the link to proposals/comment-view#single-comment?